<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterestEditWizardPage</class>
 <widget class="QWizardPage" name="InterestEditWizardPage">
  <attribute name="title">
    <string>Interest Rate</string>
  </attribute>
  <layout class="QVBoxLayout">
    <property name="margin">
      <number>11</number>
    </property>
    <property name="spacing">
      <number>6</number>
    </property>
    <item>
      <widget class="QLabel" name="TextLabel5_4">
        <property name="text">
          <string>
Please enter the interest rate or leave the field empty to calculate it.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer72">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>29</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <spacer name="Spacer77">
            <property name="sizeHint">
              <size>
                <width>101</width>
                <height>0</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
        <item>
          <layout class="QGridLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="0" column="1">
              <widget class="AmountEdit" name="m_newInterestRateEdit"/>
            </item>
            <item row="1" column="1">
              <widget class="QLabel" name="m_interestRateLabel">
                <property name="text">
                  <string>1</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="TextLabel7_3">
                <property name="text">
                  <string>Current rate</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="TextLabel6_5">
                <property name="text">
                  <string>Interest rate</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <spacer name="Spacer78">
            <property name="sizeHint">
              <size>
                <width>101</width>
                <height>0</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer73">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>29</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <widget class="QLabel" name="TextLabel9_4">
        <property name="text">
          <string>KMyMoney either calculates the new interest rate or the amount for principal and interest. If you know the interest rate then enter it here.

If KMyMoney should calculate this value for you, then leave the field blank.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
   <customwidget>
     <class>AmountEdit</class>
     <extends>QLineEdit</extends>
     <header location="global">amountedit.h</header>
   </customwidget>		 
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
